app/players/elves/elf.py

```diff
@@ -0,0 +1,15 @@
+from abc import ABC
+
+from app.players.player import Player
+
+
+class Elf(Player, ABC):

```

Comment:
u don't need to inherit from ABC one more time, u already did it because Player is inherited from ABC